### PR TITLE
Desktop Edge 2.1.2.0 relnotes

### PR DESCRIPTION
**Pull request by usha-mandya**
Release notes for Docker Desktop Edge 2.1.2.0 release.

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -4,21 +4,31 @@ keywords: Docker Desktop for Mac, edge, release notes
...
+
+[Download](https://download.docker.com/mac/edge/38030/Docker.dmg)
+
+#### Upgrades
+
+- [Docker 19.03.2](https://github.com/docker/docker-ce/releases/tag/v19.03.2)
+- [Kubernetes 1.14.6](https://github.com/kubernetes/kubernetes/releases/tag/v1.14.6)
+- [Go 1.12.9](https://github.com/golang/go/issues?q=milestone%3AGo1.12.9+label%3ACherryPickApproved)
+- [Qemu 4.0.1](https://github.com/docker/binfmt)
+- [Docker Machine 0.16.2](https://github.com/docker/machine/releases/tag/v0.16.2)
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md]: @gtardiff - Added an entry for Kitemati 0.17.8

File: docker-for-windows/edge-release-notes.md [hunk 2]
```diff
@@ -4,21 +4,31 @@ keywords: Docker Desktop for Windows, edge, release notes
...
+
+[Download](https://download.docker.com/win/edge/38030/Docker%20Desktop%20Installer.exe)
+
+#### Upgrades
+
+- [Docker 19.03.2](https://github.com/docker/docker-ce/releases/tag/v19.03.2)
+- [Kubernetes 1.14.6](https://github.com/kubernetes/kubernetes/releases/tag/v1.14.6)
+- [Go 1.12.9](https://github.com/golang/go/issues?q=milestone%3AGo1.12.9+label%3ACherryPickApproved)
+- [Qemu 4.0.1](https://github.com/docker/binfmt)
+- [Docker Machine 0.16.2](https://github.com/docker/machine/releases/tag/v0.16.2)
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-windows/edge-release-notes.md]: @gtardiff - Added an entry for Kitematic 0.17.8

**Comment by usha-mandya** [comment]
Updated the branch permissions to include:
-  at least one review before merging
- dismiss previous approvals when new commits are added

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ulrich) [reply docker-for-mac/edge-release-notes.md]: Good catch!

**Comment by usha-mandya** [comment]
@stephen-turner @nebuk89 - Could one of you approve the Edge release notes PR please?

**Pull request merged**